### PR TITLE
FSF-3932 | fix: solved control metrics calculation + execution time

### PR DESCRIPTION
**Pull request by Davide64-dev**

### CODE REVIEW

File: src/evaluator/evaluator/eufs_adapter.py [hunk 1]
```diff
@@ -59,6 +66,20 @@ def __init__(self, node: rclpy.node.Node):
         )
 
         self._time_sync_.registerCallback(self.state_estimation_callback)
+    
+    def simulated_planning_callback(
```
Comment (marhcouto) [inline src/evaluator/evaluator/eufs_adapter.py:70]: Isn't simulated planning from mocker node? What is this exactly?

**Review by marhcouto — CHANGES_REQUESTED** [review]
Probably my fault for the confusion but the other points in the task were still meant to be done in this one and not a mistake. In another note, I don't think there is simulated planning from eufs, simulated planning would be from mocker node.

File: src/evaluator/evaluator/eufs_adapter.py (hunk 1 above)
Comment (Davide64-dev) [reply src/evaluator/evaluator/eufs_adapter.py:70]: I used that nomenclature to be consistent with the remaining nodes. The idea was just to state that at the beginning of control computation. I'll fix that

**Review by marhcouto — CHANGES_REQUESTED** [review]
LGTM, just add the same feature for the pose and velocity estimation

**Comment by Davide64-dev** [comment]
> LGTM, just add the same feature for the pose and velocity estimation

What feature? I'm confused

**Review by marhcouto — APPROVED** [review]

**Pull request merged**